app/stac_api/management/commands/perftest.py

```diff
@@ -0,0 +1,118 @@
+import datetime
+import logging
+import random
+
+from dateutil.parser import isoparse
+
+from django.contrib.gis.geos import Polygon
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.core.management.base import BaseCommand
+
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import Item
+from stac_api.validators import MEDIA_TYPES
+
+# path definition relative to the directory that contains manage.py
+logger = logging.getLogger(__name__)
+
+XMIN = 2570000
+# this would be the max extent for CH
+# XMAX = 2746000
+XMAX = 2580000
+YMIN = 1130000
+# this would be the max extent for CH
+# YMAX = 1252000
+YMAX = 1140000
+
+MIN_DATETIME = isoparse('1970-01-01T00:00:01Z')
+MAX_DATETIME = isoparse('2020-12-31T23:59:10Z')
+
+
+def random_datetime(start, end):
+    """Generate a random datetime between `start` and `end`"""
+    return start + datetime.timedelta(
+        # Get a random amount of seconds between `start` and `end`
+        seconds=random.randint(0, int((end - start).total_seconds())),
+    )
+
+
+class Command(BaseCommand):
+    help = """Populates the local test database with dummy data
+
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--populate',
+            action='store_true',
+            help='Populate dummy data',
+        )
+
+        parser.add_argument(
+            '--clean',
+            action='store_true',
+            help='Clean dummy data',
+        )
+
+    def handle(self, *args, **options):
+
+        geo = Polygon.from_bbox((2570000.0, 1129970.0, 2580000.0, 1139990.0))
+        geo.srid = 2056
+        geo.transform(4326, clone=False)
+        range(2570000, 2746000, 1000)
+        range(1130000, 1252000, 1000)
+
+        if options['clean']:
+            Collection.objects.filter(name__startswith='perftest').delete()
+        elif options['populate']:
+
+            for collection_id in [f'perftest-collection-{x}' for x in range(30)]:
```

Comment:
The number of collection and asset could be added as option with a default value. e.g. `--collections=50  --assets=10`

Reply:
done